src/noise.ts

```diff
@@ -31,7 +31,7 @@ type HandshakeParams = {
 export class Noise implements INoiseConnection {
   public protocol = "/noise";
 
-  private readonly prologue = Buffer.from(this.protocol);
+  private readonly prologue = Buffer.alloc(0);
```

Comment:
is there some requirement for prologue size?

Reply:
It's not in the spec but according to Go implementation - no. https://github.com/libp2p/go-libp2p-noise/blob/master/protocol.go#L95